data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature

```diff
@@ -9217,7 +9217,7 @@ Feature: Data Import integration tests
     And headers headersUser
     And retry until response.jobExecutions[0].status == 'COMMITTED' || response.status == 'ERROR' || response.status == 'DISCARDED'
     And param fileName = "No file name"
```

Comment:
@okolawole-ebsco  I faced with issue that there can be some errors/invalid imports/etc with an empty "completed_date", just with "started_date". And in this case - the first several jobExecutions will just with "started_date", the "completed_date"-field will NOT exists there. And only after several jobExecutions with empty "completed_date" - it will provide valid sorting as we want. By sorting it via "completed_date" in desc. 
So I would like to change sorting to "started_date" - because it will exists in any case.
What do you think?

Reply:
@okolawole-ebsco  This is what I mean (several jobExecutions will be just with "started_date", "completed_date" does NOT exist):
![image](https://github.com/folio-org/folio-integration-tests/assets/42203485/dfa54faa-59d1-4f22-8a45-ab3d64b76d0e)

Reply:
I am good with using started_date!

Reply:
Thank you!